### ISSUE TITLE
$method.paramClasses shows 'final' instead of class

### ISSUE THREAD

**Issue by GoogleCodeExporter** [Bomb1369/junit-generator-v2#2]
```
If a parameter has a 'final' modifier, the 'final' modifier will be in 
'paramClasses' instead of the actual class.

```

Original issue reported on code.google.com by `peter.s....@gmail.com` on 8 Mar 2012 at 7:12

**Comment by GoogleCodeExporter** [comment]
```
Interesting. You mean 'final' on a method? Can you please post a quick example?
```

Original comment by `osborn.j...@gmail.com` on 24 May 2012 at 10:15

**Comment by GoogleCodeExporter** [comment]
```
Sure, source =

[code]
package ab;

public class JunitGeneratorIssue2 {
    public int mySum( final int a, final int b ) {
        return a + b;
    }
}
[/code]

In the default template I added '// paramClasses = $method.paramClasses' right 
after '//TODO: Test goes here...'.

Result (only relevant piece):

[code]
@Test
public void testMySum() throws Exception { 
//TODO: Test goes here...
// paramClasses = [final, final] 
} 
[/code]




```

Original comment by `peter.s....@gmail.com` on 25 May 2012 at 10:13

**Comment by GoogleCodeExporter** [comment]
```
OK. I will fix it up.
```

Original comment by `osborn.j...@gmail.com` on 27 Feb 2013 at 1:59
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Fixed the code. the original code was a little weak so I improved it and used 
the proper classes.
```

Original comment by `osborn.j...@gmail.com` on 27 Feb 2013 at 3:12
- Changed state: **Started**

**Comment by GoogleCodeExporter** [comment]
```
Released fix in build 2.0.10. please test at your convenience.
```

Original comment by `osborn.j...@gmail.com` on 5 Mar 2013 at 1:39
- Changed state: **Fixed**
